src/dblib/dblib.c

```diff
@@ -3814,13 +3824,24 @@ _get_printable_size(TDSCOLUMN * colinfo)
 	case SYBTIME:
 	case SYB5BIGTIME:
 		return 15;
+	case SYBMSTIME:
+		return 16;
 	case SYBDATE:
+	case SYBMSDATE:
 		return 10;
+	case SYBMSDATETIME2:
+		return 27;
+	case SYBMSDATETIMEOFFSET:
+		return 33;
 	case SYBUNIQUE:
 		return 36;
 	case SYBBIT:
 		return 1;
+	case SYBNUMERIC:
+	case SYBDECIMAL:
+		return colinfo->column_prec + 2;
 		/* FIX ME -- not all types present */
+	case SYBVOID: /* or 4 bytes for "null" ? */
```

Comment:
To be honest I never saw that type. I think more than `NULL` it should be used for something like `DEFAULT` calling stored procedures.